src/modules/displayShows.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] As the previous reviewer has rightly pointed, I suggest you develop the practise of not disabling linters in your codebase, but rather research for solutions that will help you solve the linter issues because linters are set up to ensure you write clean and bug-free code.

I ran your code locally and i didn't get any linter errors 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212461914-9337679a-f50c-4574-8cfb-c5ee7e8170f4.png)

Reply:
Probably you don't have ESlint plugin, you have to check for errors with the command line

Reply:
done